handlers/compute.go

```diff
@@ -42,7 +42,7 @@ func (h *ComputeHandler) InstanceActivate(event *revents.Event, cli *client.Ranc
 		return errors.Wrap(err, constants.InstanceActivateError+"failed to check whether instance is activated")
 	}
 
-	if err := compute.DoInstanceActivate(instance, host, progress, h.dockerClientWithTimeout, h.infoData); err != nil {
+	if err := compute.DoInstanceActivate(instance, host, progress, h.dockerClient, h.infoData); err != nil {
```

Comment:
Isn't switching to the no-timeout client for isntanceActivate pretty impactful? Doesn't that mean that every docker client call in instance activate will be without a timeout?